<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="Title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>721</width>
     <height>100</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:48pt;&quot;&gt;PROPERTIES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vehicle_economy_button">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>160</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Economy Vehicle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vehicle_luxury_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Luxury Vehicle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="estate_apartment_button">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>360</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Apartment Real Estate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vehicle_middle_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Vehicle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="estate_house_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>House Real Estate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="estate_penthouse_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>241</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Penthouse Real Estate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>511</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
